<launch>
  <node pkg="art_arduino" type="pid_tuning" name="arduino_pid_tuning_node" output="screen">
    <rosparam command="load" file="$(find art_arduino)/pid.yaml" />
  </node>
  <node name="reconfigure_gui_node" pkg="dynamic_reconfigure" type="reconfigure_gui"  />
</launch>
